### ISSUE TITLE
Background colour config

### ISSUE THREAD

**Issue by webmaster128** [dmester/jdenticon#24]
Hi all and thanks for doing this!

Would it be possible to add a background colour configuration like the Style.BackColor of the .net version?

> Style.BackColor (default white)
> 
> Specifies the background color of the generated icon. Set to Color.Transparent to not render any background behind the identicon shapes.

In case of Javascript, I think the default should be transparent to avoid breaking existing applications.

My current workaround is to do a string manipulation on the SVG source like this:

```javascript
// Add background to SVG
let openingRootTagEndCursor = svgSource.indexOf(">") + 1;
svgSource = svgSource.slice(0, openingRootTagEndCursor)
  + '<rect width="100%" height="100%" fill="#eeeeee" />'
  + svgSource.slice(openingRootTagEndCursor);
```

This just adds a `<rect width="100%" height="100%" fill="#eeeeee" />` as the first element inside of `<svg>`. This seems to be the most compatible way to add background to SVGs, if you do not want to move to SVG version 1.2.

**Comment by dmester** [comment]
Thanks for the suggestion, sounds like a good idea! I will look into adding an option for specifying the background color of the generated icons.

**Comment by dmester** [comment]
Hi again! Just wanted to let you know that a new option called `backColor` is now available since version 2.0.0.

**Comment by webmaster128** [comment]
Beautiful, works, thanks!
